redbeams/src/test/java/com/sequenceiq/redbeams/controller/v4/database/DatabaseV4ControllerTest.java

```diff
@@ -133,8 +133,8 @@ public void testRegister() {
     // public void testTestWithName() {
```

Comment:
Could you please delete the commented lines if you don't need for these lines anymore?

Reply:
I created a jira to work on the DatabaseConfig api part.
https://jira.cloudera.com/browse/CB-2480